WalletWasabi.Fluent/ViewModels/CoinControl/Core/ScriptType.cs

```diff
@@ -16,7 +17,7 @@ public static ScriptType FromEnum(NBitcoin.ScriptType type)
 			NBitcoin.ScriptType.P2PK => Unknown,
 			NBitcoin.ScriptType.MultiSig => Unknown,
 			NBitcoin.ScriptType.P2WSH => SegWit,
```

Comment:
```suggestion
			NBitcoin.ScriptType.P2WSH => Unknown,
```
Could be changed to Unknown as it's not supported